constants.bal

```diff
@@ -0,0 +1,174 @@
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License. 
+
+# Calls the service API V1
+public const string SERVICE_VERSION_V1 = "1.0";
+
+# Whether to include indexing directive
+public const string INDEXING_TYPE_INCLUDE = "Include";
+
+# Whether to exclude indexing directive
+public const string INDEXING_TYPE_EXCLUDE = "Exclude";
+
+# Strong consistancy level
+public const string CONSISTANCY_LEVEL_STRONG = "Strong";
+
+# Bounded consistancy level
+public const string CONSISTANCY_LEVEL_BOUNDED = "Bounded";
+
+# Session consistancy level
+public const string CONSISTANCY_LEVEL_SESSION = "Session";
+
+# Eventual consistancy level
+public const string CONSISTANCY_LEVEL_EVENTUAL = "Eventual";
+
+// Request headers
+const string CONTENT_TYPE_HEADER = "Content-Type";
+const string API_VERSION_HEADER = "x-ms-version";
+const string HOST_HEADER = "Host";
+const string ACCEPT_HEADER = "Accept";
+const string CONNECTION_HEADER = "Connection";
+const string DATE_HEADER = "x-ms-date";
+const string AUTHORIZATION_HEADER = "Authorization";
+const string THROUGHPUT_HEADER = "x-ms-offer-throughput";
+const string AUTOPILET_THROUGHPUT_HEADER = "x-ms-cosmos-offer-autopilot-settings";
+const string INDEXING_DIRECTIVE_HEADER = "x-ms-indexing-directive";
+const string IS_UPSERT_HEADER = "x-ms-documentdb-is-upsert";
+const string MAX_ITEM_COUNT_HEADER = "x-ms-max-item-count";
+const string CONSISTANCY_LEVEL_HEADER = "x-ms-consistency-level";
+const string A_IM_HEADER = "A-IM";
+const string NON_MATCH_HEADER = "If-None-Match";
+const string IF_MATCH_HEADER = "If-Match";
+const string PARTITIONKEY_RANGE_HEADER = "x-ms-documentdb-partitionkeyrangeid";
+const string ISQUERY_HEADER = "x-ms-documentdb-isquery";
+const string PARTITION_KEY_HEADER = "x-ms-documentdb-partitionkey";
+const string EXPIRY_HEADER = "x-ms-documentdb-expiry-seconds";
+const string IS_ENABLE_CROSS_PARTITION_HEADER = "x-ms-documentdb-query-enablecrosspartition";
+
+// Values for request headers
+const string CONTENT_TYPE_QUERY = "application/query+json";
+const string ACCEPT_ALL = "*/*";
+const string CONNECTION_KEEP_ALIVE = "keep-alive";
+
+// Response headers
+const string CONTINUATION_HEADER = "x-ms-continuation";
+const string SESSION_TOKEN_HEADER = "x-ms-session-token";
+const string REQUEST_CHARGE_HEADER = "x-ms-request-charge";
+const string RESOURCE_USAGE_HEADER = "x-ms-resource-usage";
+const string ITEM_COUNT_HEADER = "x-ms-item-count";
+const string RESPONSE_DATE_HEADER = "Date";
```

Comment:
Can use constants provided in ballerina http module for these kind of common headers.

Check https://ballerina.io/swan-lake/learn/api-docs/ballerina/index.html#/ballerina/http/1.0.6/http/constants#DATE